content/docs/Query.md

```diff
@@ -41,10 +105,53 @@ The prefix query finds documents containing terms that start with the provided p
 
 A fuzzy query is a term query that matches terms within a specified edit distance (Levenshtein distance).  Also, you can optionally specify that the term must have a matching prefix of the specified length.
 
+```go
+package main
+
+func main() {
+
+    index, err := bleve.Open("beer-search.bleve")
+    if err != nil {
+        log.Fatal(err)
+    }
+
+    fq := bleve.NewFuzzyQuery("Citrus") // HLBLEVE
```

Comment:
We should remove the comment `// HLBLEVE` that was added to highlight the line in a particular way in the presentation.